src/flow/layout.js

```diff
@@ -5,19 +5,25 @@ import ReactFlow, { Controls } from 'react-flow-renderer'
 
 import DAGNode from './dag-node'
 import { useElements } from './hooks'
-
+import Panel from './panel'
 
 export const nodeTypes = { DAGNode }
-const onLoad = (flow) => flow.fitView()
+const onLoad = (flow) => {
+  flow.fitView()
+}
 
 const Layout = ({ data, config, width, height }) => {
+  const panelWidth = width * 0.3
   const elements = useElements({ data, config, width, height })
-
+  
   return (
-    <div style={{ width, height }}>
-      <ReactFlow {...{ onLoad, nodeTypes, ...config }} elements={elements}>
-        <Controls />
-      </ReactFlow>
+    <div style={{ display: 'flex' }}>
+      <Panel width={panelWidth} height={height} />
+      <div style={{ width, height }}>
```

Comment:
since now we use mui styling solution, shall we leverage `makeStyles` instead of inline styles (they contribute to unnecessary performance degradations)